core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -64,6 +64,33 @@ export class PickerColumnOption implements ComponentInterface {
     this.ariaLabel = inheritedAttributes['aria-label'] || null;
   }
 
+  /**
+   * The column options can load at any time
+   * so the selected option needs to tell the
+   * parent picker column when it is loaded
+   * so the picker column can ensure it is
+   * centered in the view.
+   */
+  componentDidLoad() {
+    const parentPickerColumn = this.el.closest('ion-picker-column');
+    if (parentPickerColumn !== null && this.value === parentPickerColumn.value) {
+      parentPickerColumn.scrollActiveItemIntoView();
+    }
+  }
```

Comment:
This is new behavior. With the internal picker, the options were part of `ion-picker-column-internal`, so we knew exactly when the options were loaded. With the `ion-picker-column-option` approach, these options can be loaded at any time (such as in an `*ngIf`), so we need the options to tell the column when it is ready to be scrolled into view.